### DISCUSSION TITLE
New build for a coffee table

### DISCUSSION THREAD

**Discussion by kieraneglin** [rdudhagra/Sand-Table#33] [General]
I've been working on a fairly modified version of this project and it's been a great time! An incomplete list of changes so far:

- Modified several of the STLs to simplify their usage
- Running a BTT Manta M4P + CB1 with TMC 2209s to control the movement system
  - This removes the need for an RPI and the Arduino
- Running Klipper instead of Marlin to give me tuning tools + a web interface automatically
- Removed the back bar
- Running off a computer PSU instead of a charging brick

The LEDs are still in the mail but it looks like I can start making the table! I'll update here as I make progress. For now, I've attached a 5x timelapse of the table drawing a spiralized star pattern

https://user-images.githubusercontent.com/569917/215641979-92bdfcb2-3221-468b-bb66-f61ceae4acbe.mov

**Comment by kieraneglin** [comment]
It's coming along! I'm testing in my office with a stand-in and I'm glad I did. It let me determine a few things:

1. 1/2" mdf would work fine, but 3/8 will be better
2. I didn't realize how directional the cube magnets were. One axis in particular measured three times stronger than the other two. Makes sense, but I just didn't think about it. That explains why I was getting very inconsistent behavior when testing
3. Either 5/8 or 3/4 is the bearing size I'm going to use. In my testing it worked a lot better than smaller bearings
4. I don't love the ratio of overall footprint to usable area. To that end, I'm redesigning all the frame pieces to be as low profile as possible. I'm still testing but I think I'll be able to get an additional 50mm on the H-bar axis and 40mm on the other axis (I'll attach an example below)
5. My magnet is going to actually touch the underside of the sand area, and I'm going to use a combination of Teflon and smoothing the underside of the MDF to keep things quiet and low friction
6. I'm really enjoying using klipper for this

Smaller redesigned parts comparison:

![PXL_20230201_211309400.jpg](https://user-images.githubusercontent.com/569917/216164495-c4728c7a-b9e3-47e4-8c62-25e43a492534.jpg)


5x time lapse of it running with a bearing:

https://user-images.githubusercontent.com/569917/216162951-c6e6cfe1-1427-4326-8480-6167492e417f.mp4

**Reply by kieraneglin** [reply]
A few more pictures showcasing how I've slimmed things down. I've also taken a few millimeters off the motor mounts but I forgot to grab a photo of that. 

So far everything seems strong enough! Tomorrow I'll figure out how much space I've reclaimed

![PXL_20230201_223946537.jpg](https://user-images.githubusercontent.com/569917/216237653-b2650fe9-66b1-4fe5-9012-552ee6fd312a.jpg)

![PXL_20230201_230247002.jpg](https://user-images.githubusercontent.com/569917/216237678-1011d5de-7a68-4711-831e-514968bbf969.jpg)

![PXL_20230201_230326797.jpg](https://user-images.githubusercontent.com/569917/216237692-b8e8c3d4-0276-40e0-a6cf-edb9042a94e6.jpg)

![PXL_20230202_044515485.jpg](https://user-images.githubusercontent.com/569917/216237700-40d49281-93d2-487c-963b-345ce8c8af5b.jpg)

**Comment by kieraneglin** [comment]
After all is said and done I was able to reclaim 61mm on the H-bar and 28mm on the other axis (~2 3/8" and ~1 1/8" respectively). This is independent of extrusion length so I'll maintain those gains when I cut everything down to final size.

Not the improvement I was hoping to see on the short axis, but I still think it was worth it! I can now make the gantry area a few inches smaller while retaining the same work area 🤙 

As for strength, everything seems to be working since this is a low-speed machine anyway. The belt tension mounting bolts seem to keep everything aligned, but I'll update if something fails or comes loose eventually

**Comment by PDotStrong** [comment]
Hi NICE work! Can you share the stl file for the updated carrier without melting stuff in it? =) TY
